tests/raw_data/test_raw_data.py

```diff
@@ -0,0 +1,25 @@
+import pandas as pd
+from tests.conftest import TENANT
+from smsdk.smsdk_entities.raw_data.raw_data import RawData
+
+RAW_DATA_TABLE = "cycle_raw_data"
+NUM_ROWS = 400
+URL_V1 = "/v1/datatab/raw_data"
+
+
+def test_get_utilities(get_session):
```

Comment:
Just note that these tests are run against the demo ENV and will need to change if the env does